packages/runtimes/bun-server/run-tests.sh

```
@@ -0,0 +1,42 @@
+    --coverage)
+      coverage_mode=true
+      shift
+      ;;
+    *)
+      echo "Unknown option: $1"
+      exit 1
+      ;;
+  esac
+done
+
+files=($(find src -type f -name "*.test.ts"))
```

Shellcheck (0.11.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)